docs/_docs/editor/basics.md

```diff
@@ -228,3 +228,19 @@ The Command Palette shows you every available command available in Atom and Nucl
 You can narrow down the options that match your search by typing in the text box at the top of the Command Palette.
 
 ![](/static/images/docs/editor-basics-command-palette-search.png)
+
+## Distraction-Free Mode
```

Comment:
We should think about whether this stands alone as its own feature (i.e., that would live on the docs sidebar). I am not convinced either way right now.